UPGRADE-2.7.md

```diff
@@ -477,3 +477,10 @@ PropertyAccess
 
         new UnexpectedTypeException($value, $path, $pathIndex);
    ```
+
+Config
+------
+
+ * The `__toString()` method of the `\Symfony\Component\Config\ConfigCache` is marked as 
```

Comment:
Sure about deprecations? There are a lot of similar comments in UPGRADE-2.7 already.